packages/generate-pdf-template-helpers/src/translate.ts

```diff
@@ -1,25 +1,94 @@
-import { Translation, Translations } from './types';
+import { ExternalStandardsTranslations, Languages } from '@s1seven/schema-tools-types';
 
-export class Translate<T = Translations, t = Translation> {
-  constructor(readonly translations: T, readonly languages: string[] = ['EN']) {}
+import { Translations } from './types';
 
-  getField<G extends keyof t = keyof t, P extends keyof t[G] = keyof t[G]>(language: string, group: G, phrase: P) {
+type ValueOf<T> = T[keyof T];
+
+export class Translate<T = Translations, E = ExternalStandardsTranslations> {
+  constructor(readonly translations: T, readonly extraTranslations: E, readonly languages: Languages[] = ['EN']) {}
+  /*
```

Comment:
I added this comment to help me understand the typing but it should probably be removed before going to production.

Reply:
ok i'll remove it